Makefile

```diff
@@ -81,15 +81,17 @@ image-apiserver:
 		-t ${REGISTRY}/apiserver-$(GOARCH):$(VERSION) \
 		--platform=linux/$(GOARCH) \
 		--build-arg BASEIMAGE=$(BASEIMAGE) \
-		--build-arg BINNAME=apiserver .
+		--build-arg BINNAME=apiserver . \
+		--load
 	    
 image-clustersynchro-manager: 
 	GOOS="linux" $(MAKE) clustersynchro-manager
 	docker buildx build \
 		-t $(REGISTRY)/clustersynchro-manager-$(GOARCH):$(VERSION) \
 		--platform=linux/$(GOARCH) \
 	    --build-arg BASEIMAGE=$(BASEIMAGE) \
```

Comment:
There seems to be a wrong blank line here, could you fix it together?

Reply:
done it use 8 blank instead of tab